### DISCUSSION TITLE
Is there a way to specify just `gaps_out_top`?

### DISCUSSION THREAD

**Discussion by IgnisDa** [hyprwm/Hyprland#1315] [Q&A] [answered]
I want to apply some "padding" just to the top of the windows. Is there a config parameter that corresponds to this?

I want to display a top bar using `eww`. For now I am using `general:gaps_out` with a very high value. But I do not want to "compress" the open window through the x axis.

Thanks and regards.

**Comment by flickowoa** [comment, marked as answer]
there is no `gaps_out_top` in hyprland, tho you can use reserved space instead, iirc in  eww u can use `exclusive` to set reserved space

**Reply by deej81** [reply]
I believe this might no longer be correct? https://wiki.hyprland.org/Configuring/Variables/ gaps now support css style gaps (top, right, bottom, left -> 5,10,15,20)

**Comment by callisto1232** [comment]
Hi, is there a similar thing in waybar?

**Comment by formadi** [comment]
Apart from the gaps_out value, 
adjusting the **"addreserved value for the monitor"** allows you to increase or decrease the usable space.

monitor=,preferred,auto,1.0
**monitor=,addreserved, -100, 0, 0, 0  # top, bottom, left, right**

**Reply by cosmopolitics** [reply]
i love you

**Reply by leon-stier** [reply]
Unfortunately this no longer works as the values are clamped now. [See #12880](https://github.com/hyprwm/Hyprland/discussions/12880)
